packages/components/organisms/f-loyalty/src/tenants/en-GB.js

```diff
@@ -1,6 +1,13 @@
 const messages = {
     locale: 'en-GB',
-    text: 'I am a VLoyalty Component (GB)'
+    text: 'I am a VLoyalty Component (GB)',
```

Comment:
Will this be visible to customers?

Reply:
not atm, I will be adding things here as I build this component up, so until all the bits are built then it won't be seen by anyone